mods/d2k/rules/infantry.yaml

```diff
@@ -19,6 +19,8 @@ light_inf:
 		Speed: 43
 	Armament:
 		Weapon: LMG
+	Encyclopedia:
+		Description: Light Infantry are lightly armored foot soldiers, equipped with 9mm RP assault rifles.  Light Infantry are effective against other infantry and lightly armored vehicles.\n\nLight Infantry are resistant to missiles and large caliber guns, but very vulnerable to high explosives, fire and bullet weapons.
```

Comment:
What was your preference between single or double spacing between sentences? There's a mix throughout so whatever we go for will need amending across the files.

Reply:
I don't remember adding them on purpose, so I removed them.